lib/tasks/demo.rake

```diff
@@ -266,8 +266,11 @@ desc 'Backup the database'
         ValidationHelper::RACE_OPTIONS[exclusive ? :exclusive : :non_exclusive].map { |option| option[:race] }.sample(exclusive ? 1 : rand(0..4)).each { |race| patient[race] = true }
       end
       patient[:ethnicity] = rand < 0.82 ? 'Not Hispanic or Latino' : 'Hispanic or Latino'
-      patient[:primary_language] = rand < 0.7 ? 'English' : Faker::Nation.language
-      patient[:secondary_language] = Faker::Nation.language if rand < 0.4
+
+      available_lang_codes = Languages.all_languages.keys.to_a.map{|x| x.to_s}
```

Comment:
lets move this outside of the loop

Reply:
also, rubocop does not check demo.rake, but for styling purposes:
```suggestion
      available_lang_codes = Languages.all_languages.keys.to_a.map { |x| x.to_s }
```

Reply:
done.

Reply:
might have forgotten to do the styling change: https://github.com/SaraAlert/SaraAlert/pull/803/files#diff-9bfc38ef71b77112eb91f77e35668d17dfea639c79e19de03900b201ad44b8efR249